app/microsoft/tasks.py

```diff
@@ -53,7 +53,7 @@ def _set_up_new_user_task(user_pk: int):
 
     # Send invite email
```

Comment:
No need to do this check, we are already doing it upstream for both new and existing users.